developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,428 @@ title: "Mint an NFT with Aptos CLI"
+
+See [create_nft.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/1-Create-NFT/sources/create_nft.move) to see how to mint a token to a receiver.
+
+## 2. Use resource account for automation
+
+Since the Move model often requires knowing the signer of a transaction, Aptos provides resource accounts for assigning signer capability. Creating resource accounts enables access to the signer capability for automated use. With this ability, resource accounts can publish Move modules and automatically sign for transactions and minting. The signer capability can be assigned to the resource account or placed in storage locally within the module.
+
+When you create a resource account you also grant that account the signer capability. If we don't use a resource account for this module, we would need to manually sign for all transactions. The only field inside the signer capability is the `address` of the signer. A native function converts the signer capability to the signer.
+
+### Create a resource account
+
+Just as you created a default (typical) account on Aptos testnet, you will now do the same for a resource account to receive the NFT. This work maps to the demonstration in [create_nft_with_resource_account.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/2-Using-Resource-Account/sources/create_nft_with_resource_account.move).
+
+To create a resource account, once again issue `aptos init` but this time use the `--profile` argument to create a specific account, in this case `nft-receiver`.
+
+1. Create an `nft-receiver` account on Aptos testnet by running the following command.
+  ```shell
+  aptos init --profile nft-receiver
+  ```
+
+2. Receive the output:
+  ```shell
+  Configuring for profile nft-receiver
+  ```
+
+3. Follow the same steps as for the default account to select the `testnet` network and generate a new private key.
+
+### Publish the module under the resource account
+
+1. Assuming you are in your `aptos-core` source directory, navigate to the `aptos-core/aptos-move/move-examples/mint_nft/2-Using-Resource-Account` directory:
+
+```shell
+cd aptos-move/move-examples/mint_nft/2-Using-Resource-Account
+```
+
+2. Run the following CLI command to publish the module under a resource account, including your `<seed>` and `<default-account-address>`:
+
+```shell
+aptos move create-resource-account-and-publish-package --seed <seed> --address-name mint_nft --profile default --named-addresses source_addr=<default-account-address>
+```
+
+Remember, you can find this address in `.aptos/config.yaml`. For example:
+
+```shell
+aptos move create-resource-account-and-publish-package --seed 1235 --address-name mint_nft --profile default --named-addresses source_addr=a911e7374107ad434bbc5369289cf5855c3b1a2938a6bfce0776c1d296271cde
+```
+
+TODO: Find out what the `<seed>` should be for our users. How do they derive it? Make it up? With what syntax or input?
```

Comment:
most just make it up - if they really care about security they can generate a random number (usually they want to use a number that's hard to guess, so other people couldn't guess the address of the resource account before it's created), but for tutorial purpose they can just use something like 1234.

Reply:
Thanks! Make clear and removed the TODO.